canvas_exercise/shapes_game/index.js

```diff
@@ -1,36 +1,135 @@
 window.addEventListener("load", function() {
 
   function clear(ctx, width, heigt) {
+    ctx.clearRect(0, 0, width, height);
   }
 
-  function drawRandomShape(ctx, width, height) {
+  function drawTraingle(ctx, width, height, color) {
+    ctx.fillStyle = color;
+    ctx.beginPath();
+    ctx.moveTo(width,height);
+    ctx.lineTo(width+100, height+100);
+    ctx.lineTo(width, height+100);
+    ctx.fill();
+    ctx.closePath();
   }
 
-  function drawGameStartText(ctx, width, height, score) {
+  function drawRandomShape(shapes) {
+    var shape = shapes[Math.floor(Math.random()*shapes.length)];
+    var randWidth = Math.floor(Math.random() * (700 - 100)) + 100;
+    var randHeight = Math.floor(Math.random() * (650 - 100)) + 100;
+    var eKey;
+    switch(shape) {
+      case "redT":
+          clear(ctx, (canvas.width), (canvas.height));
```

Comment:
If you are doing the clear in all cases, you don't need to put that code in the switch.  Just do it before the switch statement.  Also, you don't need the ( ) around cavas.width and canvas.height.